source/features/pr-approvals-count.css

```diff
@@ -2,10 +2,21 @@
 .js-issue-row
 	.color-fg-muted
 	[href$='#partial-pull-merging'][aria-label*='approval'] {
-	/* font-size: 0; */
+	font-size: 0;
+	color: transparent !important;
+}
+
+.js-issue-row
+	.color-fg-muted
+	[href$='#partial-pull-merging'][aria-label*='approval']
+	@media
+	not
+	all
+	and
+	(min-resolution: 0.001dpcm) {
```

Comment:
This isn't a valid selector, you can't mix media queries and selectors

Reply:
I'll fix it after finding the reason of this bug